### PR TITLE
Dev: bootstrap: Give a warning when detecting $SSH_AUTH_SOCK but not use --use-ssh-agent option

### PR DESCRIPTION
**Pull request by liangxin1300**

### CODE REVIEW

File: crmsh/bootstrap.py [hunk 1]
```diff
@@ -143,7 +143,6 @@ def __init__(self):
                 COROSYNC_AUTH, "/var/lib/heartbeat/crm/*", "/var/lib/pacemaker/cib/*",
                 "/var/lib/corosync/*", "/var/lib/pacemaker/pengine/*", PCMK_REMOTE_AUTH,
                 "/var/lib/csync2/*", "~/.config/crm/*"]
-        self.use_ssh_agent = False
```
Comment (nicholasyang2022) [inline crmsh/bootstrap.py:146]: Do not abuse dynamic language features. Adding fields dynamically make code unreadable.

Comment (liangxin1300) [reply crmsh/bootstrap.py:146]: Some `crm cluster` subcommands don't have `--use-ssh-agent`, like `crm cluster remove`
I made this change to distinguish if the user used this option or not.

Comment (liangxin1300) [reply crmsh/bootstrap.py:146]: Or, we can add a flag to mark which option is from argparse while calling
```
@classmethod
    def set_context(cls, options):
        ctx = cls()
        for opt in vars(options):
            setattr(ctx, opt, getattr(options, opt))
        ctx.initialize_user()
        return ctx

```

Comment (nicholasyang2022) [reply crmsh/bootstrap.py:146]: `None` is enough for marking a field as not set.

**Review by nicholasyang2022 — APPROVED** [review]

**Pull request merged**